docs/stats/fit_statistics.rst

```diff
@@ -40,115 +40,180 @@ TODO
 
 In doing so, we are computing the likelihood ratio:
 
 .. math::
 
-    -2 \log \frac{L(n_{on},n_{off},\alpha; \mu_{sig},\mu_{bkg})}
-        {L(n_{on},n_{off};n_{on},n_{off})}
+    -2 \log \frac{L(n_{\mathrm{on}},n_{\mathrm{off}},\alpha;
+    \mu_{\mathrm{sig}},\mu_{\mathrm{bkg}})}
+    {L(n_{\mathrm{on}},n_{\mathrm{off}};n_{\mathrm{on}},n_{\mathrm{off}})}
 
 Intuitively, this log-likelihood ratio should asymptotically behave like a
 chi-square with ``m-n`` degrees of freedom, where ``m`` is the number of
 measurements and ``n`` the number of model parameters.
 
-Hence, we rewrite WStat as:
+
+Final result
+^^^^^^^^^^^^
+
+.. math::
+
+    W = 2 \big(\mu_{\mathrm{sig}} + (1 + \alpha)\mu_{\mathrm{bkg}} -
+    n_{\mathrm{on}} - n_{\mathrm{off}} - n_{\mathrm{on}}
+    (\log{(\mu_{\mathrm{sig}} + \alpha \mu_{\mathrm{bkg}}) -
+    \log{(n_{\mathrm{on}})}}) - n_{\mathrm{off}} (\log{(\mu_{\mathrm{bkg}})} -
+    \log{(n_{\mathrm{off}})})\big)
+
+
+Special cases
+^^^^^^^^^^^^^
+
+The above formular is obviously undefined if :math:`n_{\mathrm{on}}` or :math:`n_{\mathrm{off}}`
+are equal to zero. These cases are treated as follows.
+
+If :math:`n_{\mathrm{on}} = 0` the likelihood formular reads
+
+.. math::
+
+    L (0, n_{\mathrm{off}}, \alpha; \mu_{\mathrm{sig}}, \mu_{\mathrm{bkg}}) =
+    \exp{(-(\mu_{\mathrm{sig}}+\alpha \mu_{\mathrm{bkg}}))}\times
+    \frac{(\mu_{\mathrm{bkg}})^{n_{\mathrm{off}}}}{n_{\mathrm{off}}
+    !}\exp{(-\mu_{\mathrm{bkg}})},
+
+WStat is derived by taking 2 times the negative log likelihood as ever
+
+.. math::
+
+    W = 2 \big(\mu_{\mathrm{sig}} + (1 + \alpha)\mu_{\mathrm{bkg}} -
+    n_{\mathrm{off}} \log{(\mu_{\mathrm{bkg}})}\big)
+
+The analytical result for :math:`\mu_{\mathrm{bkg}}` in this case reads 
 
 .. math::
 
-    W = 2 \big(\mu_{sig} + (1 + \alpha)\mu_{bkg} - n_{on} - n_{off}
-    - n_{on} (\log{(\mu_{sig} + \alpha \mu_{bkg}) - \log{(n_{on})}})
-    - n_{off} (\log{(\mu_{bkg})} - \log{(n_{off})})\big)
+    \mu_{\mathrm{bkg}} = \frac{n_{\mathrm{off}}}{\alpha + 1}
+
+When inserting this into the WStat formular and also adding the goodness of fit
+term :math:`2 n_{\mathrm{off}}(\log{(n_{\mathrm{off}})} - 1)` one arrives at
+
+.. math::
 
+    W = 2\big(\mu_{\mathrm{sig}} + n_{\mathrm{off}} \log{(1 + \alpha)}\big)
+
+
+If :math:`n_{\mathrm{off}} = 0` Wstat becomes
+
+.. math::
 
-The above formular is obviously undefined if :math:`n_{on}` or :math:`n_{off}`
-are equal to zero. This case is treated as follows.
+    W = 2 \big(\mu_{\mathrm{sig}} + (1 + \alpha)\mu_{\mathrm{bkg}} -
+    n_{\mathrm{on}} \log{(\mu_{\mathrm{sig}} + \alpha \mu_{\mathrm{bkg}}
+    )}\big)
 
-If :math:`n_{on} = 0`
+and
 
 .. math::
 
-    W = 2\big(\mu_{sig} - n_{off} \log{\left(\frac{1}{1 + \alpha}\right)}\big)
+    \mu_{\mathrm{bkg}} = \frac{n_{\mathrm{on}}}{1+\alpha} -
+    \frac{\mu_{\mathrm{sig}}}{\alpha}
 
-Otherwise, two cases are distinguished.
+Obviously, :math:`\mu_{\mathrm{bkg}}` can become negative which is unphyisical.
```

Comment:
Again, I don't like "obviously" in math texts. Either remove "obviously", or if it's quick to say where, it becomes negative, do so.